src/integrations/clearml/__init__.py

```diff
@@ -0,0 +1,120 @@
+
+__all__ = [
+    'register_to_clearml',
+    'get_clearml_dataset'
+]
+
+import logging
+import re
+from typing import List, Optional
+
+import pandas as pd
+
+from clearml import Dataset
+import tempfile
+
+url_regex = re.compile(
+        r'^(?:http|ftp)s?://'
+        r'(?:(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}\.?)|'
+        r'localhost|'
+        r'\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'
+        r'(?::\d+)?'
+        r'(?:/?|[/?]\S+)$', re.IGNORECASE)
+
+
+def get_clearml_dataset(
+    dataset_id: Optional[str] = None,
+    dataset_project: Optional[str] = None,
+    dataset_name: Optional[str] = None,
+    alias: Optional[str] = None
+):
+    """Gets a local copy of ClearML Dataset
+    Args:
+        dataset_id: Requested dataset ID
+        dataset_project: Requested dataset project name
+        dataset_name: Requested dataset name
+        alias: Alias of the dataset. If set, the 'alias : dataset ID' key-value pair
+            will be set under the hyperparameters section 'Datasets'
+    Example:
+        The example shows how to get a local copy of ClearML Dataset by dataset_id
+        >>> get_clearml_dataset(dataset_id=dataset_id)
+        ...
+    """
+    if not (all([dataset_project, dataset_name]) or dataset_id):
+        raise ValueError("Either dataset_project and dataset_name or dataset_id should be provided")
+
+    dataset = Dataset.get(
+        dataset_id=dataset_id,
+        dataset_project=dataset_project,
+        dataset_name=dataset_name,
+        alias=alias
+    )
+    folder = dataset.get_local_copy()
+
+    return folder
+
+
+def register_to_clearml(
+    toloka_df: pd.DataFrame,
+    toloka_dataset_path: Optional[str] = None,
+    external_url_columns: Optional[List[str]] = None,
+    dataset_project: Optional[str] = None,
+    dataset_name: Optional[str] = None,
+    parent_dataset_ids: Optional[List[str]] = None,
+    verbose: bool = False
+) -> None:
+    """Registers a new Dataset to ClearML containing files annotated by Toloka
+    Args:
+        toloka_df: pandas Dataframe containing data from Toloka
```

Comment:
Please describe ways of obtaining this data
```suggestion
        toloka_df: pandas Dataframe containing data from Toloka. Can be obtained using `TolokaClient.get_assignments_df` or using web UI.
```